cedar-policy-validator/src/str_checks.rs

```diff
@@ -45,34 +47,31 @@ impl std::fmt::Display for ValidationWarning<'_> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         write!(
             f,
-            "Warning: {} in policy with ID {}",
+            "warning: {} in policy with ID {}",
             self.kind, self.location
         )
     }
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Error)]
```

Comment:
Any reason not to add `Eq`?
```suggestion
#[derive(Debug, Clone, PartialEq, Error, Eq)]
```